fhir-notification/src/main/java/com/ibm/fhir/notification/FHIRNotificationEvent.java

```diff
@@ -73,12 +69,4 @@ public Resource getResource() {
     public void setResource(Resource resource) {
         this.resource = resource;
     }
-
-    public HttpHeaders getHttpHeaders() {
-        return this.httpHeaders;
-    }
-
-    public void setHttpHeaders(HttpHeaders httpHeaders) {
-        this.httpHeaders = httpHeaders;
-    }
```

Comment:
need to understand why the http headers were put into the notification event, maybe the design was to keep all info in the request header including browser agent info, cookies and with all the fhir headers for better analytics later ...

Reply:
IMHO the notification stuff needs some significant reworking.  With some effort I could add the HttpHeaders back in, but IMHO they don't really belong.

Reply:
If we have a flow that takes and HTTP request and ultimately lands it on a Kafka queue (for example), then some of the contextual HTTP request information may be valuable downstream.

Reply:
I prefer to leave it off for now.  Per our discussion on Friday, I think we agreed that is OK.  If not, please re-open.